ton/wallet/integration_test.go

```diff
@@ -326,6 +349,9 @@ func TestWallet_DeployContractUsingHW3(t *testing.T) {
 }
 
 func TestWallet_TransferEncrypted(t *testing.T) {
+	if _seed == "" {
+		t.Skip()
```

Comment:
t.Fatal(emptyWalletSeedEnvFatalMsg)
```suggestion
		t.Fatal(emptyWalletSeedEnvFatalMsg)
```